src/OrchardCore.Modules/OrchardCore.Users/Views/LoginFormCredentials.cshtml

```diff
@@ -23,7 +23,7 @@
 
 <script at="Foot">
     togglePassword = document.querySelector('#togglePassword');
-    password = document.querySelector('#Password');
+    password = document.querySelector('#@Html.IdFor(m => m.Password)');
```

Comment:
Please add the directive `let` before password and togglePassword. These are local variables

Reply:
What are the difference except the intellisense, both have "#Password" value

Reply:
The old hard-coded ID that the script depended on was `Password` but the new one generated by `asp-for="Password"` is `LoginForm_Password`. This is why the script was broken.